src/rust/src/owi_sym.rs

```diff
@@ -17,29 +22,32 @@ mod sys {
     }
 }
 
-pub fn stop_exploration() -> ! {
-    unsafe {
-        sys::assume(false);
-        std::process::abort()
-    }
-}
-
+#[deprecated(
+    note = "*_symbol functions have been deprecated in favor of the Symbolic trait, use instead: `u8::symbol()`."
+)]
 pub fn u8_symbol() -> u8 {
-    unsafe { sys::i8_symbol() }
+    Symbolic::symbol()
 }
 
+#[deprecated(
+    note = "*_symbol functions have been deprecated in favor of the Symbolic trait, use instead: `u32::symbol()`."
+)]
 pub fn u32_symbol() -> u32 {
-    unsafe { sys::i32_symbol() }
+    Symbolic::symbol()
 }
 
+#[deprecated(
+    note = "*_symbol functions have been deprecated in favor of the Symbolic trait, use instead: `f32::symbol()`."
+)]
 pub fn f32_symbol() -> f32 {
-    unsafe { sys::f32_symbol() }
+    Symbolic::symbol()
 }
 
+#[deprecated(
+    note = "*_symbol functions have been deprecated in favor of the Symbolic trait, use instead: `char::symbol()`."
+)]
 pub fn char_symbol() -> char {
-    let x = u32_symbol();
-    let c = char::from_u32(x).unwrap_or_else(|| stop_exploration());
-    c
+    Symbolic::symbol()
```

Comment:
I would vote for removing all of them now. It is not as if they were used a lot and we would break any existing workflow... WDYT?